### DISCUSSION TITLE
I didn't like Linrock nets (seems unstable to me and not stronger than previous ones both tactically or positionally)

### DISCUSSION THREAD

**Discussion by microsmeta** [official-stockfish/Stockfish#4348] [General]
Tried all new nets by this author and returned to nn-ad9b42354671.nnue 

I've a Core i5 4 cores 2 Mn/s supporting AVX2 and BMI2. In overall Stockfish doesn't scale well with top hardware. On Playchess.com a Raspberry-P3 79 kN/s s can easily draw on the same version running on a 80 MN/s ThreadRipper, Xeon or a Ryzen 9 up to 128 threads and Dragon chess 3.2 has almost the same strength of latest Stockfish-dev!!! Too many draws, you must make it more aggressive  (add style settings and don't  progressively disable Classical for NNUE exclusivity!).

Thank you anyway for your hard work. You are the best, I'm a fan and very grateful to the whole SF team that changed chess engines world forever!

**Comment by vondele** [comment]
Could you clarify how you establish 'not stronger' (contrary to fishtest results 
<img width="944" alt="image" src="https://user-images.githubusercontent.com/4202567/213677490-91367962-a0de-4300-965d-380bb6d76fe5.png">
), 'unstable' (contrary to our testing: https://tests.stockfishchess.org/tests/view/63c97bd16e54ef7467d8b65b), 'scaling well' (contrary to our scaling tests https://github.com/official-stockfish/Stockfish/wiki/Useful-data#threading-efficiency-and-elo-gain), and aggressive (which is a  bit vague)

**Reply by microsmeta** [reply]
HI Vondele... It's certainly stronger on your Fishtests but when you play 5 0 using my modest hardware with right settings (eg Move overhead 300 Slow mover 100)  there is not big difference on results. All engines from PI 3 to Threadripper 128 threads in a fork 2900-3000 ELO. Why?

![Playchess](https://user-images.githubusercontent.com/8478683/213679532-20794ac4-2fac-4437-b90e-1aba92232a13.png)

I'm not here to criticize you. Thank you very much!
